src/web/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:alpine AS builder
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+COPY . /app
+RUN yarn install --frozen-lockfile
```

Comment:
Out of curiosity its there a reason why we are using yarn and not npm?

Reply:
I dont really have a reason. It is mostly because I copied the Dockerfile from nextjs https://nextjs.org/docs/deployment#docker-image

Reply:
I have moved back to npm.